lib/tape.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * Codemod for transforming Tape tests into AVA.
+ */
+
+const tapeToAvaAsserts = {
+	fail: 'fail',
+	pass: 'pass',
+
+	ok: 'truthy',
+	true: 'truthy',
+	assert: 'truthy',
+
+	notOk: 'falsy',
+	false: 'falsy',
+	notok: 'falsy',
+
+	error: 'ifError',
+	ifErr: 'ifError',
+	iferror: 'ifError',
+
+	equal: 'is',
+	equals: 'is',
+	isEqual: 'is',
+	strictEqual: 'is',
+	strictEquals: 'is',
+
+	notEqual: 'not',
+	notStrictEqual: 'not',
+	notStrictEquals: 'not',
+	isNotEqual: 'not',
+	doesNotEqual: 'not',
+	isInequal: 'not',
+
+	deepEqual: 'deepEqual',
+	isEquivalent: 'deepEqual',
+	same: 'deepEqual',
+
+	notDeepEqual: 'notDeepEqual',
+	notEquivalent: 'notDeepEqual',
+	notDeeply: 'notDeepEqual',
+	notSame: 'notDeepEqual',
+	isNotDeepEqual: 'notDeepEqual',
+	isNotEquivalent: 'notDeepEqual',
+	isInequivalent: 'notDeepEqual',
+
+	skip: 'skip',
+	throws: 'throws',
+	doesNotThrow: 'notThrows'
+};
+
+const unsupportedTestFunctions = new Set([
+	// No equivalent in AVA:
+	'timeoutAfter',
+	'deepLooseEqual',
+	'looseEqual',
+	'looseEquals',
+	'notDeepLooseEqual',
+	'notLooseEqual',
+	'notLooseEquals'
```

Comment:
We could maybe show a message that `t.deepEqual` is stricter, but might still work, but has to be applied manually.